kratos/tests/utilities/test_geometrical_sensitivity_utility.cpp

```diff
@@ -148,16 +141,19 @@ void TestThisGeometry(Geometry<Point>& rGeom,
         const Matrix& rDN_De = DN_De[g];
         GeometricalSensitivityUtility geom_sensitivity(rJ, rDN_De);
 
-        for (unsigned i_node = 0; i_node < rGeom.size(); ++i_node)
-            for (unsigned i_coord = 0; i_coord < rGeom.WorkingSpaceDimension(); ++i_coord)
-            {
-                double detJ_deriv;
-                geom_sensitivity.CalculateSensitivity(i_node, i_coord, detJ_deriv, DN_DX_deriv);
-                CheckDeterminantOfJacobianSensitivityByFiniteDifference(
-                    detJ_deriv, g, i_node, i_coord, rGeom, ThisMethod, StepSize, Tolerance);
-                CheckShapeFunctionsGradientSensitivityByFiniteDifference(
-                    DN_DX_deriv, g, i_node, i_coord, rGeom, ThisMethod, StepSize, Tolerance);
-            }
+        auto s = ShapeParameter::Sequence(rGeom.PointsNumber(),
```

Comment:
It is the sequence of shape parameters spanning `rGeom.PointsNumber()` nodes with `rGeom.WorkingSpaceDimension()` parameters per node. In 2D it basically goes (0,0), (0,1), (1,0), (1,1), (2,0), ... The reason I added it instead of simply using a double for loop is to express intent better, make the interfaces more precisely typed and reduce function parameters.